modules/sizing-requirements-for-gitops.adoc

```diff
@@ -13,13 +13,13 @@ Every time you install the {gitops-title} Operator, the resources on the namespa
 [cols="2,2,2,2,2",options="header"]
 |===
 |Workload |CPU requests |CPU limits |Memory requests |Memory limits
-|argocd-application-controller |1 |2 |1024Mi |2048Mi
-|applicationset-controller |1 |2 |512Mi |1024Mi
-|argocd-server |0.125 |0.5 |128Mi |256Mi
-|argocd-repo-server |0.5 |1 |256Mi |1024Mi
-|argocd-redis |0.25 |0.5 |128Mi |256Mi
-|argocd-dex |0.25 |0.5 |128Mi |256Mi
-|HAProxy |0.25 |0.5 |128Mi |256Mi
+|argocd-application-controller |250m |2m |1024Mi |2048Mi
```

Comment:
There are change of values, such as from 1 cpu to 250m, and in some other lines,  in addition to the format changes. Are these intended, if so where is the source requesting this change?

Reply:
Hi @chengfang, these values are taken from direct installation (in August) as @Srivaralakshmi and @Dhruv-Soni11 wanted to maintain consistency in this table and not mention values in decimal.

Reply:
Incorporated the changes as requested.